CHANGELOG.rst

```diff
@@ -1,5 +1,45 @@
 Changelog
 =========
+1.14.0 (2021-02-18)
+-------------------
+MLflow 1.14.0 includes several major features and improvements:
+
+Support for tensor inputs using the TF serving format:
+
+- Add support for TF serving input to REST API (#3894, @wentinghu)
+- Add support for np array-based tensors for Keras, PyTorch and TensorFlow flavors (#3808, @wentinghu)
+- Add tensor input support for ONNX and gluon flavors (#4041, @tomasatdatabricks)
+- Add support for tensor-based model signatures in schema enforcement (#4084, @wentinghu)
+
+For more information, see https://mlflow.org/docs/latest/models.html#deploy-mlflow-models
+
+More features and improvements:
+
+- Add SHAP Explainer logging integration (#3989, @vivekchettiar)
+- Enable skinny client package build & release (#4049, @eedeleon)
+- Update client to pass request headers created by all ``RequestHeaderProviders`` (#4042, @jimmyxu-db)
+- Use ``SavedModel`` format by default in ``mlflow.keras.log_model`` (#4043, @harupy)
```

Comment:
```suggestion
- ``mlflow.keras.log_model`` now saves models in ``SavedModel`` format by default instead of XYZ (#4043, @harupy)
```

Maybe reword to this? What was the old model format?